packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -89,7 +107,7 @@ export class NextCookies extends Cookies {
       this.response.headers.append('set-cookie', currentCookie)
     }
 
-    return store
+    return this
   }
   delete = (key: any, options: CookieSerializeOptions = {}) => {
```

Comment:
I think `key` should be a string:

```suggestion
  delete = (key: string, options: CookieSerializeOptions = {}) => {
```